chronoline/chronoline.js

```diff
@@ -394,13 +398,47 @@ function Chronoline(domElement, events, options) {
         // DRAWING
         t.floatingSet = t.paper.set();
         t.sectionLabelSet = t.paper.set();
+
+        //attach background click events function
+        t.myCanvas.onclick = function (e) {
+            //if not event or section click.
+            if (e.target.nodeName != 'circle' && e.target.nodeName != 'rect') {
+                var clickDate = new Date(t.pxToMs(e.clientX));
+                clickDate.stripTime();
+                t.backgroundClick(clickDate);
```

Comment:
hmm, i can see how that would be a complication. well, if anyone wants it in the future, it woudl be just as much work then to plop that as another argument, so it wouldn't break compatibility. It's probably fine as is.